src/multiplots.ts

```diff
@@ -1942,15 +1942,16 @@ export class MultiplePlots {
                 this.mouse_move_pp_communication();
                 this.mouse_move_frame_communication();
                 this.refreshRubberBands();
-                this.updateSelectedPrimitives();
+                // TODO: not so beautiful but here to avoid selecting points with unlinked graph points
+                if ( !(this.objectList[this.last_index] instanceof newGraph2D) ) this.updateSelectedPrimitives();
```

Comment:
You should set `this.objectList[this.last_index]` in a variable since you use it several times, here

Reply:
Done